src/Core/Rendering/RenderController.cpp

```diff
@@ -754,6 +759,44 @@ namespace MxEngine
 
         std::swap(inputOutput, temporary0);
     }
+
+    void RenderController::ApplyLensFlare(CameraUnit& camera, TextureHandle& input, TextureHandle& temporaryHalf0, TextureHandle& temporaryHalf1, TextureHandle& temporary1)
+    {
+        MAKE_SCOPE_PROFILER("RenderController::ApplyLensFlare()");
+        if (camera.LensFlare == nullptr)
+            return;
+
+        //Todo: support chromatic aberration 
```

Comment:
btw, we already have chromatic abberration effect as part of camera effects. You can steal code from there)